drivers/hopwa_caper/app/controllers/hopwa_caper/cells_controller.rb

```diff
@@ -13,11 +15,11 @@ def show
       @cell = @report.valid_cell_name(params[:id])
       @table = @report.valid_table_name(params[:table])
       @enrollments = @report.hopwa_caper_enrollments.
-        preload(:enrollment).
+        preload(:enrollment, { enrollment: :project }).
```

Comment:
Isn't this the same as the following?
```suggestion
        preload(enrollment: :project).
```

Did you intend?
```
preload(enrollment: { enrollment: :project})
```

Reply:
I believe you're right. Double checking before adding it to the PR

Reply:
added [here](https://github.com/greenriver/hmis-warehouse/pull/5203/commits/b1e0355cd6a6b7c93f77f0c4026cbcbd615f9ddc)